themes/bootstrap3/less/components/search.less

```diff
@@ -547,6 +547,9 @@ table.search-history-table {
     width: 20%;
   }
 }
+.search-history-table .history_time {
```

Comment:
@EreMaijala, it looks like subsequent to this PR being opened, you eliminated the hard-coded 20% in a different way (see the lines of code directly above this). I don't think we need both solutions to be implemented. Do you have a preference for either keeping your existing solution or adopting the one proposed here?

Reply:
@demiankatz I'm fine with either solution, but the other needs to be removed. :) The one proposed here is maybe a bit cleaner. Regarding the even/odd shading, I actually preferred the old one, but I suppose it's more down to a personal preference.